sdk/python/feast/registry_server.py

```diff
@@ -580,6 +582,17 @@ def Refresh(self, request, context):
     def Proto(self, request, context):
         return self.proxied_registry.proto()
 
+    def SetDecisionStrategy(
```

Comment:
IMO, for consistency we should also expose a Get method, even if it can be internally implemented using ListProjectMetadata